locksmith/src/controllers/v2/subscriptionController.ts

```diff
@@ -16,7 +16,7 @@ export class SubscriptionController {
     const userAddress = normalizer.ethereumAddress(request.user!.walletAddress)
 
     const subscriptions: Subscription[] =
-      await subscriptionOperations.getSubscriptionsForLockByOwner({
+      await subscriptionOperations.getSubscriptionForLockByOwner({
```

Comment:
Since you are retuning an array, why do you rename?

Reply:
related to this old comment https://github.com/unlock-protocol/unlock/pull/11840#discussion_r1189996104

Reply:
I see then it should *not* return an array? There is a single `Subscription` object for a specific `keyId`, right?